koji/collection-mash-split.py

```diff
@@ -420,17 +413,12 @@ def main():
         else:
             git_tag = "rpm/{}".format(version)
 
-            if LooseVersion(version) == LooseVersion('2.0'):
-                dists['fedora29'] = 'fc29'
-
         mashes = [MashConfig(collection, version, dist, dist, code) for dist, code in dists.items()]
     elif collection == 'katello':
         branch_map = {
             'nightly': 'develop',
             '3.18': '2.3',
             '3.17': '2.2',
-            '3.16': '2.1',
```

Comment:
There is still a last 3.16.2 planned (https://community.theforeman.org/t/release-meeting-notes-2020-12-02/21509).

Reply:
now that 3.16.2 is released, are we still blocked on this?

Reply:
I don't think so. @ianballou can you confirm that there will be no more 3.16 releases?